langchain/document_loaders/directory.py

```diff
@@ -36,6 +37,7 @@ def __init__(
         loader_kwargs: Union[dict, None] = None,
         recursive: bool = False,
         show_progress: bool = False,
+        use_multithreading: bool = False,
```

Comment:
@PawelFaron Thank you for the contribution! :heart: 

What do you think about an option to limit the number of threads / concurrency?

Reply:
Yes I was also thinking about it. Will do.

Reply:
I've set it to 4. Or maybe you have some other suggestion?